src/Routes/RosPage/RosPage.js

```diff
@@ -299,11 +269,10 @@ class RosPage extends React.Component {
                                 className: 'ros-systems-table'
                             }}
                             variant="compact"
-                            hideFilters={{ all: true, name: false }}
+                            hideFilters={{ all: true, name: false, operatingSystem: false }}
```

Comment:
@r14chandra since we are using the OS filter provided by inventory, it will also show the filter options for RHEL versions not supported by ROS as shown in the below screenshot. Have you confirmed with Pau/Stefan about the same? is it okay if we show the unsupported versions from the UX point of view?

More info here: https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/assessing_and_monitoring_rhel_resource_optimization_with_insights_for_red_hat_enterprise_linux/proc-ansible-install_ros-install#proc-ros-install_ros-install